pkgs/applications/audio/neothesia/default.nix

```diff
@@ -0,0 +1,93 @@
+{ lib
+, rustPlatform
+, fetchFromGitHub
+, cli ? false
+, pkg-config
+, cmake
+, glib
+, alsa-lib
+, atkmm
+, gtk3
+, wrapGAppsHook
+, ffmpeg-full
+, vulkan-loader
+, makeWrapper
+, fontconfig
+, noto-fonts
+, libclang
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "neothesia" + (if cli then "-cli" else "");
+  version = "unstable-2023-09-23";
+
+  src = fetchFromGitHub {
+    owner = "PolyMeilex";
+    repo = "Neothesia";
+    rev = "d724865059e876f14a3b92ba18a1c5dbe3485131";
+    hash = "sha256-CnVfv0iCuUgZcmtpxg8+wgG7r8HsLdKgjCbbNu5o76c=";
+  };
+
+  strictDeps = true;
+  nativeBuildInputs = [
+    pkg-config
+    cmake
```

Comment:
What is cmake actually doing here? I can't tell that it gets used

Reply:
hmm, I don't remember anymore why I put it there but I didn't get any errors when building without it so I guess it's not needed.

Reply:
not getting any errors without atkmm anymore either. pretty sure it was needed at some point.